Fix bug on history when resource's title is too long

ElviaBth opened the pull request:
#### :tophat: What? Why?
Fix "History" feature, since layout breaks when the title of the affected resource (e.g. a proposal) exceeds 70 characters.

#### :pushpin: Related Issues
- Related to #?
- Fixes #14564 

#### Testing

1. Open a development app, go to the site
2. Go to "Processes", "Assemblies" or "Conferences"
3. Click "Proposals" on the participatory space navigation sidebar menu.
4. Click on one of the Proposals listed there.

### :camera: Screenshots
![image](https://github.com/user-attachments/assets/46b3116f-5ffb-4206-a898-87bcdf745f87)
History on desktop view
![image](https://github.com/user-attachments/assets/71e97594-e545-4944-bd26-693114ca16de)
History on mobile view

:hearts: Thank you!

ElviaBth commented:
> @ElviaBth thanks for fixing this! Now it looks great on mobile. Can you please adjust the desktop view as well?
Hi @carolromero. This PR is ready for a re-review. The new design for the desktop was applied.

carolromero reviewed (dismissed):
@ElviaBth re-reviewed and approved, thanks for your work!

alecslupu reviewed (changes requested):
invesigating this fix, i have added a proposal with more than 70 chars. after that i have added it to a budget ( import proposals to budget in admin ) . 
The history has been created ok, but in budget, the history is not being displayed corectly: 
![image](https://github.com/user-attachments/assets/ff6eca16-afb8-48c1-9201-550514bd117d)

@ElviaBth could you have a new look on it ?

ElviaBth commented:
> invesigating this fix, i have added a proposal with more than 70 chars. after that i have added it to a budget ( import proposals to budget in admin ) . The history has been created ok, but in budget, the history is not being displayed corectly
> @ElviaBth could you have a new look on it ?

Hi @alecslupu, applied changes, now it is ready for re-check.

alecslupu reviewed (changes requested):
checking the tablet ( or any smart phone that rotated can give a resolution of around 950 px) ... I am thinking that if you change the breakpoints from `lg` to `md` should do the trick: 
![image](https://github.com/user-attachments/assets/f4ab9b5f-0c88-455a-8480-9e4972970b8f)

alecslupu reviewed (approved):
:+1: Better than what we have now. it could still be improved. Will let it pass, as it is an improvement.

alecslupu commented:
Adding `no-backport` as this addresses a functionality unavailable in 0.29.
